pkg/kubelet/pleg/generic.go

```diff
@@ -406,6 +406,9 @@ func (g *GenericPLEG) updateCache(pod *kubecontainer.Pod, pid types.UID) error {
 	// all containers again.
 	status, err := g.runtime.GetPodStatus(pod.ID, pod.Name, pod.Namespace)
 	if err != nil {
+		// nolint:logcheck // Not using the result of klog.V inside the
+		// if branch is okay, we just use it to determine whether the
+		// additional parameter should be used.
```

Comment:
Don't understand the commend about `additional parameter`

Reply:
That refers to the `"podStatus", status` key/value pair which only gets included at higher log levels. Comment updated.